### PR TITLE
chore: grid ui improvements

### PR DESCRIPTION
**Pull request by richardshiue**
adjust text style, padding and icons

### Feature Preview

---

#### PR Checklist

- [x] My code adheres to [AppFlowy's Conventions](https://docs.appflowy.io/docs/documentation/software-contributions/conventions)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

### CODE REVIEW

File: frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart [hunk 1]
```diff
@@ -15,23 +16,30 @@ class GridAddRowButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final color = Theme.of(context).brightness == Brightness.light
+        ? const Color(0xFF171717).withOpacity(0.4)
+        : const Color(0xFFFFFFFF).withOpacity(0.4);
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart:21]: In the codebase, there are 4+ similar definitions. All of them are using the same color but with different opacities.

My original idea was to refactor them with `context.xxxColor.withOpacity`, however, in some cases, the opacities are even different in light mode and dark mode.

File: frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart [hunk 2]
```diff
@@ -15,23 +16,30 @@ class GridAddRowButton extends StatelessWidget {
...
+    final borderColor = Theme.of(context).isLightMode
+        ? const Color(0xFFEDEDEE)
```
Comment (LucasXu0) [inline frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart:23]: the value of `theme.lightBorderColor` is `Color(0xFFEDEDEE)`

File: frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart (hunk 1 above)
Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart:21]: I agree with you, this doesn't look good at all. I can't wait for clearer color definitions

File: frontend/resources/flowy_icons/16x/add.svg
Comment (LucasXu0) [inline frontend/resources/flowy_icons/16x/add.svg:1]: The icon is a basic icon. We need to double check the UI reference for it.

File: frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart (hunk 2 above)
Comment (richardshiue) [reply frontend/appflowy_flutter/lib/plugins/database/grid/presentation/widgets/footer/grid_footer.dart]: Oh nice. Done

File: frontend/resources/flowy_icons/16x/add.svg
Comment (richardshiue) [reply frontend/resources/flowy_icons/16x/add.svg:1]: I renamed the new one to add_less_padding and restored the old one

**Pull request merged**